sechub-wrapper-owasp-zap/src/test/java/com/mercedesbenz/sechub/zapwrapper/scan/ZapScannerTest.java

```diff
@@ -343,7 +343,7 @@ void set_includes_and_excludes_api_facade_is_called_once_for_each_include_and_on
 
         /* test */
         verify(clientApiFacade, times(includes.size())).addIncludeUrlPatternToContext(any(), any());
-        verify(clientApiFacade, times(includes.size())).accessUrlViaZap(any(), any());
+        verify(clientApiFacade, times(2)).accessUrlViaZap(any(), any());
```

Comment:
why is this a 2 and not as formerly the included size. Please at least document why this magic number is used (or maybe I just do not get it...)